lib/streams/germanAbbreviationStream.js

```diff
@@ -1,20 +1,45 @@
-var _ = require('lodash');
 var through = require('through2');
```

Comment:
Just to keep things clear it would be great to rename this file to `germanicAbbreviationStream`. A small change but an important one for clarity.